erpnext/stock/doctype/item/item.json

```diff
@@ -1020,6 +1021,12 @@
    "fieldname": "website_image_alt",
    "fieldtype": "Data",
    "label": "Image Description"
+  },
+  {
+   "default": "1",
```

Comment:
This shouldn't be enabled by default right? @sagarvora @barredterra

Reply:
Previously, this was applicable for all items. This keeps it backwards compatible. Alternative is patch with default as 0. But I think that there are generally lesser no. of items where commission isn't granted.

Reply:
If you think it is getting undue attention because the checkbox defaults to 1, we can change the field to be **Don't Grant Commission** instead.